src/LoaderController.res

```diff
@@ -322,6 +330,13 @@ let make = (~children, ~paymentMode, ~setIntegrateErrorError, ~logger, ~initTime
             })
             logger.setClientSecret(clientSecret)
 
+            // Update useTopRedirectionAtom
```

Comment:
The above change and this change is same do we need both? or if we need can we make a func and call?